### PR TITLE
[MJAVADOC-578] Groups parameter is not compatibile with Surefire

### PR DESCRIPTION
**Pull request by eolivelli**
See https://issues.apache.org/jira/browse/MJAVADOC-578 for reference

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java [hunk 1]
```diff
@@ -1043,7 +1043,7 @@
      * See <a href="./apidocs/org/apache/maven/plugin/javadoc/options/Group.html">Javadoc</a>.
      * <br/>
      */
-    @Parameter( property = "groups" )
+    @Parameter()
```
Comment (rfscholte) [inline src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java]: And no need for parentheses anymore

Comment (eolivelli) [reply src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java]: @rfscholte 
sure, thank you, removed

**Comment by eolivelli** [comment]
@rfscholte I think we don't need an integration test for this change

**Comment by eolivelli** [comment]
Link to CI results
https://builds.apache.org/job/maven-box/job/maven-javadoc-plugin/job/MJAVADOC-578

**Comment by rfscholte** [comment]
> I think we don't need an integration test for this change

I agree. Removing an impossible user property to use should have no effect.

**Comment by eolivelli** [comment]
Good.
I will merge as soon as I have a "blue" build on Jenkins

**Pull request closed without merging**